src/main/java/com/hartwig/platinum/pdl/ConvertFromGCSPaths.java

```diff
@@ -28,34 +32,34 @@ public static List<TumorNormalPair> apply(final List<SampleConfiguration> config
             if (!sample.tumors().isEmpty()) {
                 for (RawDataConfiguration tumor : sample.tumors()) {
                     if (tumor.bam().isPresent()) {
-                        pairs.add(ImmutableTumorNormalPair.builder()
-                                .reference(sample.normal().map(n -> ImmutableSample.builder().name(n.name()).bam(n.bam()).build()))
-                                .tumor(ImmutableSample.builder()
+                        pairs.add(ImmutablePipelineInput.builder()
```

Comment:
It's a bit of a shame to use `ImmutablePipelineInput.builder()` when you created `PipelineInput.builder()` for this exact use case I think :) Same for all the other builders.